Examples/Device/Device.Thermometer.nanoFramework/Device.Thermometer.nanoFramework.nfproj

```diff
@@ -26,28 +26,42 @@
     <ProjectReference Include="..\..\..\nanoFramework\Amqp.nanoFramework.nfproj" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="mscorlib">
```

Comment:
here's the reason the app.config: there a duplicate entry for mscorlib

Reply:
Removed